deployment/terraform/strings.go

```diff
@@ -549,7 +549,12 @@ RestartSec=1
 WantedBy=multi-user.target
 `
 
-const otelcolOperatorAppAgent = `
+const otelcolOperatorApp = `
+      - type: move
+        from: body.MESSAGE
+        to: body`
+
+const otelcolOperatorAgent = `
```

Comment:
I'm kinda regretting this decision of writing this modularly instead of just writing a long string of text for each :joy: